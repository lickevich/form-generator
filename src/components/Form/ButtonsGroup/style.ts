import styled from "styled-components";

export const StyledButtonsGroup = styled.div`
  display: flex;
  justify-content: flex-end;
  flex-wrap: wrap;
  padding: 20px;
`;
