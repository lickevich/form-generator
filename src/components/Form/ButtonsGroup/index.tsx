import React from "react";

import { StyledButtonsGroup } from "./style";

const ButtonsGroup: React.FC<React.ReactNode> = ({ children }) => (
  <StyledButtonsGroup>{children}</StyledButtonsGroup>
);

export default ButtonsGroup;
