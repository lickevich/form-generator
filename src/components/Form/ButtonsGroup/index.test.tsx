import React from "react";
import { render } from "@testing-library/react";

import ButtonsGroup from "./";

describe("ButtonsGroup", () => {
  it("render the component", () => {
    const component = (
      <ButtonsGroup>
        <input type="button" value="OK" />
        <input type="reset" value="RESET" />
      </ButtonsGroup>
    );
    const { asFragment } = render(component);

    // @ts-ignore
    expect(asFragment(component)).toMatchSnapshot();
  });
});
