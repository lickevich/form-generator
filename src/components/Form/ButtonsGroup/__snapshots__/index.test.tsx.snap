// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonsGroup render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvaa gZJYjV"
  >
    <input
      type="button"
      value="OK"
    />
    <input
      type="reset"
      value="RESET"
    />
  </div>
</DocumentFragment>
`;
